cli/store_test.go

```diff
@@ -39,7 +39,7 @@ func TestStore(t *testing.T) {
 		t.Fatalf("unable to save cluster: %v", err)
 	}
 
-	if err = st.ValidateName(cluster.Name); err != ErrAlreadyExists {
+	if err = st.ValidateName(cluster.Name); err.Error() != ErrAlreadyExists {
```

Comment:
Using `testify/assert` could be a nice way to avoid having to call `err.Error()` (which will panic if err is nil, instead of failing the test)

Reply:
Agreed. Will do in another PR.